tests/integration/main-test.js

```diff
@@ -0,0 +1,177 @@
+    });
+
+    //This adapter is used to use ember local storage instead of firestore
+    this.owner.register('adapter:application', class TestService extends Adapter {
+   
+    });
+  });
+
+  //This makes the database empty after the end of the test
+  hooks.afterEach(function() {
+    if (window.localStorage) {
+      window.localStorage.clear();
+    }
+    if (window.sessionStorage) {
+      window.sessionStorage.clear();
+    }
+    resetStorages();
+  });
+
+  test('Main flow of the application', async function(assert) {
+
+    //These three asserts prove that an unauthorized user can´t see the content of the pages and are redirected
+    //to login form
+    await visit('admin/invitations');
+    assert.equal(currentURL(), '/login', 'Assert unauthorized route admin/invitations');
+
+    await visit('admin/contacts');
+    assert.equal(currentURL(), '/login', 'Assert unauthorized route admin/contacts');
+
+    await visit('admin/seeder');
+    assert.equal(currentURL(), '/login', 'Assert unauthorized route admin/seeder');
+
+    //Sending an invitation. Also check button is disabled with no e-mail written and not disable with valid
+    //email
+    await visit('');
+    assert.dom('[data-test="saveInvitation"]').isDisabled();
+    await fillIn('[data-test="emailField"]', 'cool@cool.com');
+    await pauseTest();
+    assert.dom('[data-test="saveInvitation"]').isNotDisabled();
+    await click('[data-test="saveInvitation"]');
+    await pauseTest();
+
+    //Sending a contact message. Also check button is disabled without valid e-mail and message that got less than
+    //five characters. After filled in the right requirements check the button is not disabled
+    await visit('contact');
+    assert.dom('[data-test="saveContact"]').isDisabled();
+    await fillIn('[data-test="contactEmail"]', 'hellofrommars@gmaioul.com')
+    assert.dom('[data-test="saveContact"]').isDisabled();
+    await fillIn('[data-test="contactMessage"]', 'Hello!! Greetings from Mars!!')
+    await pauseTest();
+    assert.dom('[data-test="saveContact"]').isNotDisabled();
+    await click('[data-test="saveContact"]');
+    await pauseTest();
+
+    //Visiting admin/invitations without login so when the login is done you are redirected to
+    //admin/invitations
+    await visit('admin/invitations');
+    
+    //Authenticating the session with ember-simple-auth/test-support
+    await authenticateSession();
+    await pauseTest();
+    
+    //Asserting that you are redirected to the last atempted route after login (authenticateSession())
+    assert.equal(currentURL(), '/admin/invitations', "Confirm redirected after authenticateSession");
+    assert.equal(this.element.querySelector('h1').textContent, 'Invitations', "Confirm is in invitation page");
+
+    //Asserting that the invitation was stored in local database and is show in the page
+    assert.equal(find('[data-test="invitationEmail"]').textContent, 'cool@cool.com',
```

Comment:
@tpetrone  I can´t get what you mean. He is comparing the value in the table with the cool@cool.com that was inserted in the saveinvitation form...... If I change the cool@cool.com to sss for example it will not pass.